### DISCUSSION TITLE
MySQL Data Source - Save and test function

### DISCUSSION THREAD

**Discussion by walberts** [grafana/grafana#52599] [Data Sources]
![image](https://user-images.githubusercontent.com/31254820/180001946-9a94abe1-28e2-47a7-bf14-3cccef337aa8.png)

**What happened**:
When creating a new data source connection with MySQL the Save and test button returns a "Validation error, invalid URL: """. Also editing existing data source connections exhibits the same issue.

**What you expected to happen**:
A save of the connection and then a report if the connection works or fails. It worked in previous versions.

**How to reproduce it (as minimally and precisely as possible)**:
Create a new MySQL data source.
**Anything else we need to know?**:
The default Host:localhost:3306 does not get passed as the host for the data source. Manually entering it makes the connection work.

**Environment**:
- Grafana version: v9.0.3 (023f9251a9)
- Data source type & version:
- OS Grafana is installed on: Windows 10
- User OS & Browser: Firefox, Chrome
- Grafana plugins: MySQL
- Others:

**Comment by usmangt** [comment]
This seems more of a question with the MySQL connection case and nothing to do with Grafana. 

It will be more helpful to view the log lines inside `/var/log/grafana/grafana.log`

**Comment by gabor** [comment]
hi @walberts , as we got no response since @usmangt 's recommendation for a long time, we'll close this discussion. if you think there's a bug in grafana, please open a github issue describing your problem in more detail:
- do you have the URL field filled out?
- could you try in the newest grafana version?
- what was the grafana version where this worked?
